examples/extensions/r2/hello_r2.py

```diff
@@ -35,6 +35,7 @@ def my_sandbox(path, rootfs):
     ql.hook_address(func, r2.functions['main'].offset)
     # enable trace powered by r2 symsmap
     # r2.enable_trace()
+    r2.bt(0x401906)
```

Comment:
Why do we need an extra argument?

Reply:
The extra argument is the target address for setting the bt_hook, so we can print backtrace without `emu_error()`. I will test `_backtrace_fuzzy` without argument in emu_error()